.circleci/config.yml

```diff
@@ -1,122 +1,77 @@
 version: 2
 jobs:
-  make-eksctl-image:
-    # use machine build, as otherwise we cannot use make and have to duplicate makefile
+  test-and-build:
     machine:
       image: ubuntu-1604:201903-01
     environment:
-        GO_VERSION: 1.12.6
-        GOCACHE: /home/circleci/.cache/go-build/
-        GOPATH: /home/circleci/go
+      BUILD_IMAGE: weaveworks/eksctl-build:19ff80e010d2fdb7be3c44fc6491e13395a682ad
```

Comment:
Nit: do you know if there is a way to factorise `BUILD_IMAGE: weaveworks/eksctl-build:19ff80e010d2fdb7be3c44fc6491e13395a682ad` in order to have it defined at a higher-level in the YAML file?